### ISSUE TITLE
Toasts should disappear on unmount

### ISSUE THREAD

**Issue by Da-Colon** [decent-dao/fractal-interface#422]
There are toasts that are not set to close on success that are not closing on unmount. For Example if you Delegate and start the transaction and then navigate away you notice that the toast will remain.

**Comment by adamgall** [comment]
Will the toast eventually close once the transaction completes, regardless of where the user has navigated to?

**Comment by Da-Colon** [comment]
You are right. I'm gonna remove the bug label but not close just yet. I have seen this somewhere during the QA I did a week ago and just remembered about it. I'll follow up and close this if this was an older bug with routing.

**Comment by Da-Colon** [comment]
Can no longer reproduce, closing issue
